### PR TITLE
[author] [ajv] updated bundles

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: ajax/libs/ajv/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     {
       "basePath": "dist",
       "files": [
-        "ajv.*"
+        "*.*"
```
Comment (PeterDaveHello) [inline ajax/libs/ajv/package.json]: What about just `"*"` for all files not naming with a starting `.` (dot)? Or `"**/*"` for recursive including.

**Comment by PeterDaveHello** [comment]
Hi @epoberezkin,
Thanks for your PR, would you please rebase your branch of this PR with our latest master branch? The parent of this PR's commit was a little bit too old. For easier way, you can just create another PR by going to https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/ajv/package.json can press the edit(pancil) button, and then commit, send the PR, thanks for your contribution again.

**Comment by epoberezkin** [comment]
@PeterDaveHello thanks, will do it this way

**Comment by PeterDaveHello** [comment]
#6927

**Pull request closed without merging**